src/test/java/org/wildfly/security/ssl/MechanismDatabaseTest.java

```diff
@@ -43,4 +48,20 @@ public void testEdhDheMapping() {
         System.out.println(entry);
     }
 
+    @Test
+    public void testAllJdkCipherSuitesMapping() {
+        final MechanismDatabase mechanismDatabase = MechanismDatabase.getInstance();
+
+        SSLServerSocketFactory ssf = (SSLServerSocketFactory) SSLServerSocketFactory.getDefault();
+        String[] supportedCipherSuites = ssf.getSupportedCipherSuites();
+        List<String> TLS1_3CipherSuites = Arrays.asList("TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384");
```

Comment:
I think it's safer to list all five of the TLS 1.3 cipher suites here (the ones that are actually available are dependent on the JDK version).

Reply:
fixed